cmd/konvoy-image/cmd/gcp.go

```diff
@@ -0,0 +1,42 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+
+	"github.com/mesosphere/konvoy-image-builder/pkg/app"
+)
+
+var (
+	gcpExample = "gpc ... images/azure/centos-7.yaml"
+	gcpUse     = "gcp <image.yaml>"
+)
+
+var gcpBuildCmd = &cobra.Command{
+	Use:     gcpUse,
+	Short:   "build and provision gcp images",
+	Example: gcpExample,
+	Args:    cobra.ExactArgs(1),
+	Hidden:  true,
+	Run: func(cmd *cobra.Command, args []string) {
+		runBuild(args[0])
+	},
+}
+
+func initBuildGCP() {
+	fs := gcpBuildCmd.Flags()
+
+	initGenerateFlags(fs, &buildFlags.generateCLIFlags)
+	initGCPFlags(fs, &buildFlags.generateCLIFlags)
+
+	addBuildArgs(fs, &buildFlags)
+}
+
+func initGCPFlags(fs *flag.FlagSet, gFlags *generateCLIFlags) {
+	gFlags.userArgs.Azure = &app.AzureArgs{}
```

Comment:
I am sure you want to add gcp flags here. :) . but wiring wise it looks ok. 
I think you can add the gcp subcommand and flags without hiding and fake/empty business logic. 
Next set of PRs can fill those logic. wdyt?